document/Introduction.mdk

```diff
@@ -26,9 +26,9 @@ or conformance to certification standards.
 
 * Adopt a widely used serialization format that can be parsed by readily available tools. 
 
-* Represent issues that occur in all kinds of programming artifacts, including source code and object code. 
+* Represent static analysis results for all kinds of programming artifacts, including source code and object code. 
 
-* Represent the logical construct within which an issue is detected, such as a function, class, or namespace. 
+* Represent the logical construct within which a result is produced, such as a function, class, or namespace. 
 
-* Represent the physical location where an issue is detected, including issues that are detected in nested files
+* Represent the physical location where a result is produced, including problems that are detected in nested files
```

Comment:
@michaelcfanning Changed this one to "at which". I can see producing a result "against a class" -- there's something wrong with the class. I don't like it as much here. If I produce a result at line 12, it's not the "line" _per se_ that's faulty, it's the code on that line -- so I like "at which" better.